docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -46,6 +46,8 @@ core|default|role|Option values: `Mixed/Receiver/Aggregator`. **Receiver** mode
 | - | - | maxSizeOfAnalyzeProfileSnapshot| The maximum number of snapshots analyzed by the OAP. | - | 12000 |
 | - | - | prepareThreads| The number of threads used to prepare metrics data to the storage. | SW_CORE_PREPARE_THREADS | 2 |
 | - | - | enableEndpointNameGroupingByOpenapi | Automatically groups endpoints by the given OpenAPI definitions. | SW_CORE_ENABLE_ENDPOINT_NAME_GROUPING_BY_OPAENAPI | true |
+| - | - | maxDurationOfAnalyzeEBPFProfiling| The maximum duration(in minute) of analyze the eBPF profiling data. | - | 10 |
+| - | - | maxDurationOfQueryEBPFProfilingData| The maximum timeout duration(in second) of query the eBPF profiling data when analysis. | - | 30 |
```

Comment:
`maximum` and `timeout` is duplicated :) Choose one. There is not a minimal timeout, right?

Reply:
And according to your codes, this is related to time of the database query, right?

Reply:
Yes, `maxDurationOfAnalyzeEBPFProfiling` means the user chooses the time range max duration. `maxDurationOfQueryEBPFProfilingData ` means the profiling data query to storage max duration. the second one uses to avoid users waiting for the query taking a long time.

Reply:
```suggestion
| - | - | maxDurationOfQueryEBPFProfilingData| The maximum duration(in second) of query the eBPF profiling data from database. | - | 30 |
```

I mean this.